Pythonwin/Scintilla/src/LexGen.py

```diff
@@ -201,7 +201,9 @@ def FindProperties(lexFile):
 
 
 def ciCompare(a, b):
-    return cmp(a.lower(), b.lower())
+    a_lower = a.lower()
```

Comment:
this code is "vendored", so I don't really want to fork it. I don't think it's used in practice by pywin32.

Reply:
I meant to revert all scintilla-related code (see your other similar comment about Scintilla). Missed this one, thanks!